bosch_thermostat_client/sensors/sensors.py

```diff
@@ -60,7 +60,7 @@ class Sensors(BoschEntities):
     """Sensors object containing multiple Sensor objects."""
 
     def __init__(
-        self, connector, sensors_db={}, uri_prefix=None, data=None, parent=None
+        self, connector, sensors_db: dict | None = None, uri_prefix=None, data=None, parent=None, errors: dict | None = None
```

Comment:
Just curious, what's the idea behind the `sensors_db` change from `{}` to `None`?

Reply:
It will get evaluated once only and not for every call.

More details (for a list, but it's the same for a dict) here https://stackoverflow.com/questions/64136035/why-should-i-set-a-function-list-argument-as-empty-or-none-instead-of-using-a-no

Reply:
Ouch, I had forgotten about this rough edge of Python. Good improvement 👍